LayoutTests/imported/w3c/web-platform-tests/css/css-scroll-snap/snap-to-visible-areas-both-psuedo.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<title>
+  Snap to a visible area only even when there is a closer snap point for an area
```

Comment:
Is this title (and the text below) accurate for what's being tested here? It looks like this was copied from another test.

Reply:
Yes this is equivalent to the other test except the two snap areas are pseudo elements. Added this at the end to make it clear.